native-to-erc20/contracts/contracts/test/BridgeValidatorsMock.sol

```diff
@@ -1,11 +1,11 @@
 pragma solidity 0.4.24;
 
-import '../upgradeable_contracts/native_to_erc20/ForeignBridgeValidators.sol';
+import '../upgradeable_contracts/BridgeValidators.sol';
 
-contract ForeignBridgeValidatorsMock is ForeignBridgeValidators {
+contract BridgeValidatorsMock is BridgeValidators {
   using SafeMath for uint256;
 
-  function setRequiredSignatures(uint256 _requiredSignatures) public {
+  function setRequiredSignatures(uint256 _requiredSignatures) external {
```

Comment:
Should save some gas having this as external 👍

Reply:
does it saves gas?
on the BridgeValidator if was defined `external` so needed to make it the same here

Reply:
Yes externals will save gas right? because they don't need to store the arguments because there is no internal calling. But for 256 bit it's only a tiny saving....

Reply:
@Andrew-Pohl true, thanks for noticing this :)
this explains it pretty well
https://ethereum.stackexchange.com/questions/19380/external-vs-public-best-practices